src/audio/copier/copier_dai.c

```diff
@@ -182,50 +256,19 @@ int copier_dai_create(struct comp_dev *dev, struct copier_data *cd,
 		break;
 	case ipc4_alh_link_output_class:
 	case ipc4_alh_link_input_class:
+#if defined(CONFIG_ACE_VERSION_2_0)
+		dai.type = SOF_DAI_INTEL_HDA;
```

Comment:
if I follow the same logic, then the dai.type would need to be INTEL_HDA for 

````c
case ipc4_i2s_link_output_class:
case ipc4_i2s_link_input_class:
case ipc4_dmic_link_input_class:
````

So this change in isolation is a change of convention. Either ALL LNL DAIS become INTEL_HDA, or none become INTEL_HDA.

Reply:
There was no answer to this comment from 2 months ago....

Reply:
The main open is that your PR is called "lnl: sndw: use HDA DMA to transfer stream data"

But that's not really an option, this is what the hardware requires, and not only for Soundwire but DMIC/SSP as well.

So the point is that it's still unclear why we are special-casing SoundWire when from a DMA perspective the same support is required as SSP.

It's even more confusing since we are now adding aggregation concepts in the HDaudio case, when aggregation is a concept that only exists for SoundWire.